### DISCUSSION TITLE
Question Regarding Screen Monitoring

### DISCUSSION THREAD

**Discussion by CS99175** [SafeExamBrowser/seb-win-refactoring#119] [Q&A]
I have a question regarding the Safe Exam Browser screen monitoring @dbuechel . Can proctors monitor students laptop screen via Safe Exam Browser? Is that option available yet.?

**Comment by dbuechel** [comment]
No, this functionality is not yet available, but it is on the roadmap for SEB and SEB-Server.

**Reply by CS99175** [reply]
@dbuechel I need to examine what are my students doing at the examination time on their laptop. Is there a way for me to examine their works or things that they are doing or functions they are using during the examination period if I have enable Excel functionality on Safe Exam Browser ?

**Reply by dbuechel** [reply]
You can do so to a certain degree with [SEB Server](https://github.com/SafeExamBrowser/seb-server). There however isn't an option yet to see the desktop of users, as stated above.

Another option would be to allow Zoom or another meeting software as permitted application and then use the screen sharing of that third-party application to observe your students resp. their desktops.

**Reply by CS99175** [reply]
@dbuechel You have said that I can do it in a certain degree with SEB server. So please tell me what are the option available for me to examine students works in SEB server. What are the things that I can examine through SEB server and how do I enable those features ?

**Reply by dbuechel** [reply]
You can e.g. check the application log of all connected SEB clients in real time. With the upcoming SEB Server and SEB versions, you'll also be able to use basic remote proctoring with Zoom via SEB Server. More details about these features and further information can be found in the user guide of SEB Server: https://seb-server.readthedocs.io/en/latest/monitoring.html

**Reply by CS99175** [reply]
@dbuechel  This is my final question...Please clarify me on this... I want to held an examination and permitting Excel application to answer. But student may have use pre-saved answers in their Excel sheets and use them during the examination. Is there a way that I can examine whether student have use those pre-saved Excel sheets to answer during the examination.?

**Reply by dbuechel** [reply]
No, without SEB Server or another proctoring solution you won't be able to verify this.

**Reply by CS99175** [reply]
@dbuechel  That means I can do it if I have SEB server in my hand?

**Reply by dbuechel** [reply]
No, you can't see the desktop of the users as already explained. But you can see the application log in real time, via which you should be able to e.g. read the window title of the currently opened document in Excel or see which application(s) the user has opened.

**Reply by CS99175** [reply]
Thank you very much for your support. SEB is a very good application and useful in these pandemic situation. Keep it up brother and good luck for your future endevours also.
